### ISSUE TITLE
[med] Label: Many Pages: Contact search field has no visible label, only placeholder text

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5510]
## Action Performed:
Prerequisite: user has created a personal account and is logged into same.
1. Using Chrome, open https://staging.new.expensify.com/
2. Click on the "+" menu button and select "New Group" option.
3. Notice the appearance of the "Name, email or phone number" text field.
4. Begin typing any characters into the field and check if the field label is still visible.

## Expected Result:
The field should have a persistent, visible label. Placeholder text should not be used as the sole visible label for any field.

## Actual Result:
The field uses only placeholder text as the visible label (i.e. 'Name, email or phone number'), which describes the accepted input type, but not the purpose of the field (separate issue). The placeholder text is removed from view once the user begins entering text.

HTML:
`<input placeholder="Name, email, or phone number" autocapitalize="sentences" autocomplete="on" autocorrect="on" dir="auto" spellcheck="true" type="text" class="css-11aywtz r-tz77pi" value="" style="background-color: rgb(255, 255, 255); border-color: rgb(236, 236, 236); border-radius: 8px; border-width: 1px; color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; height: 42px; padding: 10px 12px; vertical-align: middle;">`

Other occurrences:
1. The search field in search side panel on Chat page.
2. The message bar field on Chat page.
3. The "Name, email or phone number" text field in New Chat overlay.

## Workaround:
No workaround

## Area issue was found in:
New group overlay, Chat page, New chat overlay

## Failed WCAG checkpoints
3.3.2

## User impact:
User with cognitive impairments may not remember what input the field accepts and for what purpose. They will be forced to delete any text already input in order to see the placeholder text again. This makes it frustrating and difficult for users to check their input before submitting.

## Suggested resolution:
Do not use placeholder text as the only method to visually label input fields. Ensure that any label used for the field remains visible at all times. The <label> tag with "for" attribute can be used to associate the visible label with the text field.
https://www.w3.org/WAI/tutorials/forms/labels/
https://www.w3.org/WAI/tutorials/forms/instructions/#placeholder-text

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5182583_New_Chat_overlay_-_placeholder_text_is_used_as_the_only_label_for_the_field](https://user-images.githubusercontent.com/88733897/134730053-066e4509-16f5-4358-af87-bcc42e83e70b.png)
![Bug5182583_Other-Chat-Contact_search_field_has_no_visible_label_only_placeholder_text-General_Chat_Search](https://user-images.githubusercontent.com/88733897/134730059-d32db0f7-3d74-402d-920b-52605e9d9fed.png)
![Bug5182583_Other-Chat-Contact_search_field_has_no_visible_label_only_placeholder_text-Message_bar](https://user-images.githubusercontent.com/88733897/134730065-8b9b3987-b9ef-4b8c-85b1-f8ed694255a9.png)
![Bug5182583_Other-Create_New_Group-Contact_search_field_has_no_visible_label__only_placeholder_text](https://user-images.githubusercontent.com/88733897/134730076-c763c7a3-1f71-42c4-99c7-1996d9ec52a0.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
I guess an easy fix here would be to use our standard form styles where the small label floats above the input value after the user enters something?

**Comment by ogumen** [comment]
@shawnborton not sure if it was a question for me, but I'll comment it. The floating labels are a good approach for accessibility as long as they are always visible and announced by screen readers.
